File: bestia-memoryserver/src/main/java/net/bestia/memoryserver/Application.java
package net.bestia.memoryserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.ComponentScan;

/**
 * Main application entry point. This will start the spring-boot application.
 *
 * @author Thomas Felix
 */
@SpringBootApplication
@ComponentScan("net.bestia.model.dao")
public class Application {

  public static void main(String[] args) {
    SpringApplication.run(Application.class, args);
  }
}

File: bestia-memoryserver/src/main/kotlin/net/bestia/memoryserver/Application.kt
package net.bestia.memoryserver

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan

/**
 * Main application entry point. This will start the spring-boot application.
 *
 * @author Thomas Felix
 */
@SpringBootApplication
@ComponentScan("net.bestia.model.dao")
class Application

fun main(args: Array<String>) {
  SpringApplication.run(Application::class.java, *args)
}